src/components/Topology/TopologyPopover/topologySort.tsx

```diff
@@ -35,7 +35,7 @@ export function getSortLabels(topology: Topology[]) {
         labels[h.name] = true;
         currentSortLabels.push({
           id: defaultSortLabels.length + index,
-          value: (h.name ?? "").toLowerCase(),
+          value: `field:${(h.name ?? "").toLowerCase()}`,
```

Comment:
I don't think this should be lower case, if someone writes "Memory" etc. in their list this will break, also we should allow use to define custom sorting fields later on properties